assets/src/util/dateTime.ts

```diff
@@ -49,13 +49,14 @@ export const formattedTimeDiffUnderThreshold = (
     : formattedTime(b)
 }
 
-/** Takes a time of day in seconds since midnight, offset in minutes (reasonably -720 to 720)
+/** Takes a time of day in seconds since midnight, offset in seconds (reasonably -8640 to 8640 seconds, or -12 to 12 hours)
  */
 export const formattedScheduledTime = (
   time: number,
   offset?: number | undefined
 ): string => {
-  let minutes = Math.floor(time / 60) + (offset ? offset : 0)
+  const offsetInMinutes = offset ? Math.floor(offset / 60) : 0
+  let minutes = Math.floor(time / 60) + offsetInMinutes
```

Comment:
question: would the given offset ever not be a multiple of 60? Thinking there could be an edge case if we are doing `Math.floor` separately on the offset and the time. Maybe this should be `let minutes = Math.floor((time + offset) / 60)`?

Reply:
Good catch! I'm not actually sure if it's possible to do non-whole increments of minutes in the UI, though if it's easy enough to combine into one `Math.floor` call that would probably be best to safeguard us from future bugs.